docs/GDI-SOP_charter.md

```diff
@@ -0,0 +1,167 @@
+
+The governance model within the GDI network is designed to be both collaborative and flexible, **supporting the adoption of European-wide standards while accommodating the unique needs of individual nodes**. This dual approach is reflected in the structure of the GDI SOPs, which are divided into [**European-level SOPs**](../sops/european-level/) that provide standardized procedures applicable across all nodes, and [**Node-specific SOPs**](../sops/node-specific/) that allow for customization based on local requirements.
+
+SOPs will be **version controlled** in GitHub, with a single source of truth: this repository. Furthermore, through GitHub we expect to **collaboratively and transparently** develop and maintain the SOPs. By having the main repository public, yet allowing GDI nodes to privately make their implementations, we comply with the rule of thumb _as open as possible, yet as private as needed_. For further details, refer to the [GitHub management](./GDI-SOP_github-management.md) documentation.
+
+Through this Charter, the GDI underscores its commitment to fostering a unified yet adaptable operational environment, ensuring that all nodes, regardless of their current capabilities, can contribute to and benefit from this pan-European initiative.
+
+## 5. Purpose of this Charter
+The purpose of this document is to **define the scope and management of SOPs within the GDI network**. Specifically, it outlines how GDI SOPs will be managed to ensure consistency, quality, and relevance. This Charter serves as a foundational document within the GDI SOP quality management system, providing the necessary guidance and structure for the development, approval, and implementation of SOPs across the network.
+
+## 6. Scope of GDI SOPs
+The GDI is not a monolithic entity but a network of existing and emerging infrastructures. While SOPs can encompass various aspects of a team, node, or even project, this repository is not intended to cover every process within all institutions under the GDI umbrella. Instead, it contains **SOPs that are relevant to GDI-specific use cases** and are implemented by the nodes forming the GDI network.
+
+For example, if a GDI node already functions as an archive with submitted datasets, the curation of these datasets, when outside the GDI scope, is not expected to follow a hypothetical "Dataset Curation SOP" from GDI.
+
+Therefore, it is the **responsibility of the GDI nodes to implement these SOPs when the use case pertains to GDI infrastructure**, governance, or data.
+
+The scope of these SOPs is focused on roles, topics, and resources that are essential for establishing or operating node services, ensuring high-quality data management, and maintaining robust technical infrastructure. However, certain items are deliberately excluded from the current scope to maintain focus.
+
+### In-Scope
+#### Roles
```

Comment:
The type of people concerned is much more important than what they are called (we know that peoples titles vary a lot) so I suggest writing like this instead:

- People responding to and resolving user queries and issues within the node (e.g. Helpdesk Staff and Operations Bioinformaticians)
- People focused on the preparation, harmonization, and curation of data and metadata to ensure high-quality data management (e.g. Data Stewards, Data Curators, and Bioinformaticians).
- People tasked with designing, developing, implementing, deploying, and monitoring software and services deployed at a node (e.g. Software Developers, Software Engineers, and DevOps Engineers)